src/block/divider/editor.scss

```
@@ -0,0 +1,136 @@
+@import "common";
+
+@include desktop {
+	:where(.has-text-align-left) .stk-block-divider {
+		--stk-dots-margin-left: 0;
+		--stk-dots-margin-right: auto;
+	}
+
+}
+
+@include desktop {
+	:where(.has-text-align-right) .stk-block-divider {
+		--stk-dots-margin-right: 0;
+		--stk-dots-margin-left: auto;
+	}
+}
+
+@include desktop {
+	:where(.has-text-align-center) .stk-block-divider {
+		--stk-dots-margin-right: auto;
+		--stk-dots-margin-left: auto;
+	}
+}
+
+@include tablet {
+	:where(.has-text-align-left-tablet) .stk-block-divider {
+
+		--stk-dots-margin-left: 0;
+		--stk-dots-margin-right: auto;
+	}
+}
+
+@include tablet {
+	:where(.has-text-align-right-tablet) .stk-block-divider {
+
+		--stk-dots-margin-right: 0;
+		--stk-dots-margin-left: auto;
+	}
+}
+
+@include tablet {
+	:where(.has-text-align-center-tablet) .stk-block-divider {
+		--stk-dots-margin-right: auto;
+		--stk-dots-margin-left: auto;
+	}
+
+}
+
+@include mobile {
+	:where(.has-text-align-left-mobile) .stk-block-divider {
+		--stk-dots-margin-left: 0;
+		--stk-dots-margin-right: auto;
+	}
+}
+
+@include mobile {
+	:where(.has-text-align-right-mobile) .stk-block-divider {
+		--stk-dots-margin-right: 0;
+		--stk-dots-margin-left: auto;
+	}
+}
+
+.stk-block.stk-block-divider {
+
+	@include desktop {
+		&.has-text-align-center {
+			--stk-dots-margin-right: auto;
+			--stk-dots-margin-left: auto;
+		}
```

Please combine the styles for each responsive size to make them easier to read.
```suggestion
		&.has-text-align-center {
			--stk-dots-margin-right: auto;
			--stk-dots-margin-left: auto;
		}
		&.has-text-align-right {
			--stk-dots-margin-right: 0;
			--stk-dots-margin-left: auto;
		}
```